internal/handlers/common/tail.go

```diff
@@ -0,0 +1,47 @@
+package common
+
+import (
+	"github.com/FerretDB/FerretDB/internal/pg"
+	"github.com/FerretDB/FerretDB/internal/types"
+)
+
+// Tail creates a tail for sql query
```

Comment:
We should clarify that, and add a trailing dot

Reply:
hm, should I add more information to this? add some - that it's for count and find queries (not for everything)